server/queue/queue-rabbitmq/src/test/java/org/apache/james/queue/rabbitmq/MailDTOTest.java

```diff
@@ -46,6 +46,7 @@
 import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
 
 class MailDTOTest {
+    static final EnQueueId EN_QUEUE_ID = EnQueueId.ofSerialized("110e8400-e29b-11d4-a716-446655440000");
```

Comment:
EN_QUEUE_ID is not consistent with what is written elsewhere.
BTW, enqueue should be a single word, so no EnQueue and EN_QUEUE but Enqueue and ENQUEUE, right?